cypress/e2e/visual-regression.cy.js

```diff
@@ -0,0 +1,48 @@
+describe('Visual Regression Testing', () => {
+  let timestamp;
+
+  before(() => {
+    cy.log('Clearing screenshots before starting the test');
+    cy.task('clearScreenshots');
+
+    timestamp = new Date().toISOString().replace(/[:.]/g, '-');
+    cy.log(`Timestamp for this run: ${timestamp}`);
+
+    cy.log('Visiting the work.co page');
+    cy.visit('https://work.co');
+
+    cy.log('Checking for Allow Cookies button');
+    cy.get('button').contains('Allow Cookies').then(($btn) => {
+      if ($btn.length) {
+        cy.log('Found Allow Cookies button, attempting to click');
+        $btn.click({ force: true });
+        cy.log('Clicked Allow Cookies button successfully');
+      } else {
+        cy.log('Allow Cookies button not found, proceeding without clicking');
+      }
+    });
+
+    cy.log('Capturing baseline screenshot');
```

Comment:
you can have this line if (Cypress.env('createBaseline')) ... if true we save screenshots in baseline folder,and if false we compare
and then use npx run cypress:run --env createBaseline=true or npm run cypress:run --env createBaseline=false for running the test